### DISCUSSION TITLE
Access Mealie off network

### DISCUSSION THREAD

**Discussion by dgaz9** [mealie-recipes/mealie#1113] [Q&A] [answered]
Hello! I just installed Mealie and love it! I would like to use this off network as well and share with my family.

What is the best way to securely open this up to the public Internet?

**Comment by zierbeek** [comment, marked as answer]
The standard way to set this up is a VPN or get a reverse proxy.
The reverse Proxy is discribed in thé docs. Swag is An option. You can use nginx or nginx Proxy manager
https://docs.mealie.io/documentation/community-guide/swag/
